### PR TITLE
Portuguese translation

### PR DESCRIPTION
**Pull request by arthuroliveirasou**

### CODE REVIEW

File: index.pt.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                href="https://jaehyuk-lee.github.io/KartriderRushPlus_SeasonPassCalc/index.cn">中文繁體</a></span>
+    </p>
+    <ul>
+        <li>Tradução: <span class="link-effect"><a
+                    href="https://linktr.ee/KartRiderBR"
+                    target="_blank">Comunidade Brasileira de KartRider</a></span>
+            por<span class="link-effect"><a
+                    href="https://linktr.ee/ArthurOliveiraSou"
+                    target="_blank">Arthur Oliveira</a></span>
+        <li>email do desenvolvedor: alex00728@outlook.com</li>
```
Comment (Jaehyuk-Lee) [inline index.pt.html]: Could you add description about "I don't know Portuguese. Use English please." thing, under the email address.
Example: Chinese page have that. `（我不懂中文。請用英文。English Please.）` (*I don't know Chinese. Use English.* English Please.)
https://jaehyuk-lee.github.io/KartriderRushPlus_SeasonPassCalc/index.cn

**Review by Jaehyuk-Lee — COMMENTED** [review]
Thanks for the PR.
I couldn't imagine this webpage was actually viewed overseas. I thought KRRP died except Korea.
I like Kartrider series and I hoped Kartrider to spread worldwide, that was why I translated this in English 😃  I'm surprised there's dedicated community in Brazil.
I have one suggestion.

File: index.pt.html (hunk 1 above)
Comment (arthuroliveirasou) [reply index.pt.html]: Okay. I will...

**Comment by Jaehyuk-Lee** [comment]
Thanks for the PR!

**Comment by Jaehyuk-Lee** [comment]
one link have wrong url, 
did you meant https://linktr.ee/KartRiderBR as https://linktr.ee/KartRiderBrasil ?

**Comment by arthuroliveirasou** [comment]
> 
> 
> one link have wrong url,
> did you meant https://linktr.ee/KartRiderBR as https://linktr.ee/KartRiderBrasil ?

Yes, the correct one is / KartRiderBrasil.
I confused with the user on instagram

**Pull request merged**